includes/class-wc-stripe-intent-controller.php

```diff
@@ -404,14 +404,16 @@ public function update_payment_intent( $payment_intent_id = '', $order_id = null
 		if ( $payment_intent_id ) {
 
 			$request = [
-				'amount'   => WC_Stripe_Helper::get_stripe_amount( $amount, strtolower( $currency ) ),
-				'currency' => strtolower( $currency ),
-				'metadata' => $gateway->get_metadata_from_order( $order ),
+				'amount'      => WC_Stripe_Helper::get_stripe_amount( $amount, strtolower( $currency ) ),
+				'currency'    => strtolower( $currency ),
+				'metadata'    => $gateway->get_metadata_from_order( $order ),
+				'description' => __( 'Stripe - Order', 'woocommerce-gateway-stripe' ) . ' ' . $order->get_id(),
```

Comment:
Looks like the shortcode flow uses the site title (instead of "Stripe") and the order number (instead of order ID) [[source](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/ace1b9f93a950a62d20d194ce44a74286e99ed2b/includes/abstracts/abstract-wc-stripe-payment-gateway.php#L421)] – should it be the same here? Could open an issue for that if so